python/tank/bootstrap/manager.py

```diff
@@ -23,6 +23,12 @@ class ToolkitManager(object):
     and installations.
     """
 
+    # Used to indicate that the manager is bootstrapping the toolkit (with method _bootstrap_sgtk).
+    TOOLKIT_BOOTSTRAP_PHASE = "sgtk"
```

Comment:
nitpick: do we need the string representations? What about an enum-style representation?

```
(STARTUP_PHASE_BOOTSTRAP, STARTUP_PHASE_ENGINE) = range(2)
```

This is a pattern we tend to use in other places.